unity-ggjj/Assets/Scripts/NarrativeScript/NarrativeScriptPlaylist/INarrativeScriptPlaylist.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+
+public interface INarrativeScriptPlaylist
+{
+    public void InitializeNarrativeScripts();
+    
+    NarrativeScript GetNextNarrativeScript();
+    IEnumerable<NarrativeScript> GetAllNarrativeScripts();
+    NarrativeScript GetRandomFailureScript();
```

Comment:
Super mega ultra nit: I think grouping these methods by return type makes things a tiny bit easier to read.
```suggestion
    NarrativeScript GetNextNarrativeScript();
    NarrativeScript GetRandomFailureScript();
    IEnumerable<NarrativeScript> GetAllNarrativeScripts();
```